test/napi/src/js/symbols.rs

```diff
@@ -0,0 +1,22 @@
+use neon::prelude::*;
+
+pub fn return_js_symbol_with_description(mut cx: FunctionContext) -> JsResult<JsSymbol> {
+    Ok(cx.symbol("neon:description"))
+}
+
+pub fn return_js_symbol(mut cx: FunctionContext) -> JsResult<JsSymbol> {
+    Ok(JsSymbol::new(&mut cx))
+}
+
+pub fn read_js_symbol_description(mut cx: FunctionContext) -> JsResult<JsValue> {
+    let symbol: Handle<JsSymbol> = cx.argument(0)?;
+    match symbol.description(&mut cx) {
```

Comment:
FYI, this can also be done with combinators.

```suggestion
    symbol
        .description()
        .map(|v| Ok(v.upcast()))
        .unwrap_or_else(|| Ok(cx.undefined()))
```